nvflare/fuel/f3/cellnet/core_cell.py

```diff
@@ -460,7 +460,7 @@ def __init__(
         )
 
         self.sent_msg_size_pool = StatsPoolManager.add_msg_size_pool(
-            "Sent_Msg_Sizes", "Sizes of messages sent (MBs)", scope=self.my_info.fqcn
+            "Sent_Msg_sizes", "Sizes of messages sent (MBs)", scope=self.my_info.fqcn
```

Comment:
need to keep the original "Sent_Msg_Sizes"

Reply:
What do you mean? This "change" was done by the "black" command and I don't see any difference. Do you?